pkg/registry/rbac/clusterrole/policybased/storage.go

```diff
@@ -52,7 +52,7 @@ func (r *Storage) NamespaceScoped() bool {
 }
 
 func (r *Storage) TenantScoped() bool {
-	return false
+	return true
```

Comment:
what does this tenant scoped mean and why is it changed to true now?

Reply:
and if it's always going to be tenant-scoped, do we still need this boolean?

Reply:
This value, together with the func NamespaceScoped(), indicate the scope of a resource type:
TenantScoped(): true, NamespaceScoped: true -> NamespaceScoped
TenantScoped(): true, NamespaceScoped: false -> TenantScoped
TenantScoped(): false, NamespaceScoped: false -> ClusterScoped